stoobly_agent/app/cli/helpers/endpoints_import_service.py

```diff
@@ -124,14 +145,19 @@ def import_response_header_name(project_id: int, endpoint_id: int, response_head
     ENDPOINT_COMPONENT_NAMES[4]: import_response_param_name
 }
 
-def process_import(component_name: str, *args):
-    return component_name_import_dispatch[component_name](*args)
+def process_import(component_name: str, body_param_parents: Dict[int, int], response_param_parents: Dict[int, int], *args):
```

Comment:
Suggestion, could rename to `import_factory` and return just the import method to avoid having the pass the rest of the arguments.